packages/react-native-worklets/android/src/main/java/com/swmansion/worklets/networking/WorkletsNetworking.kt

```
@@ -0,0 +1,227 @@
+                        val code = errorCode(requestId, call, e)
+                        cleanup(requestId)
+                        listener.onError(code, e.message ?: e.javaClass.simpleName)
+                    }
+
+                    override fun onResponse(
+                        call: Call,
+                        response: Response,
+                    ) {
+                        try {
+                            response.use { handleResponse(it, useBytesResponse, listener) }
+                        } catch (t: Throwable) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)